Artesian/_Query/ActualQuery.py

```diff
@@ -1,59 +1,147 @@
+from Artesian import _ClientsExecutor
+from Artesian._ClientsExecutor import RequestExecutor
 from Artesian._Query.Query import _Query
 from Artesian._Query.QueryParameters.ActualQueryParameters import ActualQueryParameters
 from Artesian._Query.Config.ExtractionRangeConfig import ExtractionRangeConfig
 from Artesian._Query.Config.Granularity import Granularity
 from Artesian._Configuration.DefaultPartitionStrategy import DefaultPartitionStrategy
 import urllib
+from typing import List
+from __future__ import annotations
+
 class _ActualQuery(_Query):
     __routePrefix = "ts"
-    def __init__(self, client, requestExecutor, partitionStrategy):
+    def __init__(self, client: _ClientsExecutor, requestExecutor: RequestExecutor, partitionStrategy: DefaultPartitionStrategy) -> None:
+        """ Inits _ActualQuery
+         
+            Args:          
```

Comment:
Args are not documented. either doucment or remove documentation.

Since this is a private-only init, removing is fine.